scripts/babel-relay-plugin/src/__fixtures__/connectionWithSelectVariables.fixture

```diff
@@ -0,0 +1,338 @@
+Input:
+var Relay = require('react-relay');
+var x = Relay.QL`
+  query {
+    node(id: 123) {
+      ... on User {
+        friends1: friends(first: $limit) {
+          edges {
+            ... on UserConnectionEdge @relay(selectVariables: []) {
+              node {
+                name
+              }
+            }
+          }
+        }
+        friends2: friends(first: $limit) {
+          edges {
+            ... on UserConnectionEdge @relay(selectVariables: ["size"]) {
```

Comment:
could we allow `.. on Type @relay(variables: {size: $size})` such that the developer gets complete control?